rules/windows/process_creation/win_bad_opsec_sacrificial_processes.yml

```diff
@@ -0,0 +1,25 @@
+title: Bad Opsec Defaults Sacrificial Processes With Improper Arguments
+id: a7c3d773-caef-227e-a7e7-c2f13c622329
+status: experimental
+description: Detects attackers using tooling with bad opsec defaults e.g. spawning a sacrificial process to inject a capability into the process without taking into account how the process is normally run, one trivial example of this is using rundll32.exe without arguments as a sacrificial process (default in CS, now highlighted by c2lint), running WerFault without arguments (Kraken - credit @am0nsec), and other examples.
```

Comment:
I believe as you are using the `@` sign in the field, you need to put quotes at the beginning and the end of the `description` field

Reply:
Thanks. Let me fix it.

Reply:
Fixed.

Reply:
Thanks!